datasource/utils.go

```diff
@@ -4,6 +4,61 @@ package datasource
 
 import "strconv"
 
+var DivisionalAreas = [3]string{"Arts and Humanities", "Social Sciences", "Science & Engineering & Applied Science"}
+
+func getGenEdInfo(intAttributes []interface{}) []string {
+
+	// Type conversion, since we're dealing with an interface.
+	var attributes []map[string]interface {}
+	for _, item := range intAttributes {
+		attrMap, ok := item.(map[string]interface {}) 
+		if !ok {
+			panic("An item in courseAttributes is not of type map[string]interface")
+		}
+		attributes = append(attributes, attrMap)
+	}
+
+	areas := []string{}
+	for _, attribute := range attributes {
+		if attribute["crseAttributeDescription"] == "FAS General Education" {
+			areas = append(areas, attribute["crseAttrValueDescription"].(string))
```

Comment:
Likewise we can just populate the code here instead of the description

Reply:
By this, do you mean to look for flags representing different gen-ed areas and populate from an array of hardcoded tags we have rather than relying on `crseAttrValueDescription` to be correct?

Reply:
I think he means read from `crseAttrValue`. I didn't know it existed until I just checked; maybe you didn't either? Its value is either `A&C`, `E&C`, `HSI`, or `STS`. Funnily, Curricle escapes the ampersands as `\u0026`, so the first two are actually `A\u0026C` and `E\u0026C` (which you probably want to un-escape).

Reply:
To save you some time, the `\u`s are part of the JSON spec and so they'll be automatically converted back to unicode